core/cc/osx/crash_handler.cpp

```diff
@@ -45,9 +50,16 @@ const char* GetTempDir() {
 } // namespace
 
 CrashHandler::CrashHandler() :
-    mHandlerFunction(defaultHandlerFunction),
-    mHandler(new google_breakpad::ExceptionHandler(
-        GetTempDir(), NULL, ::handleCrash, reinterpret_cast<void*>(this), true, NULL)) {
+    mNextHandlerID(0),
+    mExceptionHandler(nullptr) {
+
+    if (!Debugger::isAttached()) {
+        mExceptionHandler = std::unique_ptr<google_breakpad::ExceptionHandler>(
+            new google_breakpad::ExceptionHandler(
+                GetTempDir(), nullptr, ::handleCrash,
+                reinterpret_cast<void*>(this), true, nullptr));
```

Comment:
This could potentially be prevented in CrashHandler::handleMinidump by returning false, which will allow any other exception handlers (i.e. the debugger) to process the crash. I haven't tested this personally though.

Reply:
Done